addons/mail/static/tests/m2x_avatar_user_tests.js

```diff
@@ -25,7 +25,7 @@ QUnit.module('mail', {}, function () {
             Many2OneAvatarUser.prototype.partnerIds = {};
 
             Object.assign(this.data, {
-                'foo': {
+                'ir.attachment': {
```

Comment:
Fields should be in mock model. Keep foo if you need to create fake fields on attachment instead.

I was suggesting attachment for user_id (which exists but is named `create_uid`), but if it doesn't have a x2many on user_id, forget about it for now.